cmd/notary/tuf.go

```diff
@@ -1031,14 +1037,14 @@ func maybeAutoPublish(cmd *cobra.Command, doPublish bool, gun data.GUN, config *
 		return err
 	}
 
-	cmd.Println("Auto-publishing changes to", gun)
-	return publishAndPrintToCLI(cmd, nRepo, gun)
+	cmd.Println("Auto-publishing changes to", nRepo.GetGUN())
```

Comment:
I don't think you need the getter because we use the`gun` argument to get the repository [here](https://github.com/docker/notary/pull/1105/files/6a1f1c1f5a528906d486d3d0a21afef98658264f#diff-1409503b7b31865d44186c540b70fb43R1034)
```
nRepo, err := notaryclient.NewFileCachedNotaryRepository(
 		config.GetString("trust_dir"), gun, getRemoteTrustServer(config), rt, passRetriever, trustPin)
```